src/test/java/clear_all_use_case/ClearAllControllerTest.java

```diff
@@ -0,0 +1,71 @@
+package clear_all_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessEvent;
+import data_access.DataAccessTask;
+
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class ClearAllControllerTest {
+    private final static ClearAllPresenter OUTPUT_BOUNDARY = new ClearAllPresenter();
+    private final static ClearAllDataAccess DATA_ACCESS = new ClearAllDataAccess();
+    private final static ClearAllDsGateway DS_GATEWAY = DATA_ACCESS;
+    private final static ClearAllInteractor INPUT_BOUNDARY = new ClearAllInteractor(OUTPUT_BOUNDARY, DS_GATEWAY);
+    private final static ClearAllController CONTROLLER = new ClearAllController(INPUT_BOUNDARY);
+
+    @Test
+    void testClearAll() {
+        ClearAllOutputData outputData = CONTROLLER.clearAll();
+
+        assertTrue(outputData.getSuccess());
+        assertTrue(DATA_ACCESS.getDays().get(0).getTasks().isEmpty());
+        assertTrue(DATA_ACCESS.getDays().get(1).getTasks().isEmpty());
+        assertTrue(DATA_ACCESS.getDays().get(2).getTasks().isEmpty());
+
```

Comment:
Shouldn't you be checking across all 7 days of the week (i.e. from 0 to 6 inclusive) and not just the first 3 days. In addition, I think you should make all the days have some Task and Event, since the days attribute by default contains Day objects which have no Tasks and Event.

Reply:
Added the other 4 days with tasks, events and tested them as well.